client/ayon_houdini/startup/otls/ayon_lop_import.hda/ayon_8_8Lop_1lop__import_8_81.0/Help

```diff
@@ -0,0 +1,16 @@
+= AYON Load Asset =
+
+#icon: path/to/icon
+
+"""References a USD file, usually an asset."""
+
+== Overview ==
+
+*References* or *Payloads* a USD file into the current USD layer.
+
+In USD, when referencing, it's good to understand that you load a *single root prim* from the USD file. You will not get the full contents of the USD file if the USD file's content do not live within that single root primitive. By default, it will load a USD file's *default prim* but can be customized using the Load Options on this LOP node. 
+
```

Comment:
what do you think about ?
```suggestion
"""References an AYON USD product, usually an asset or a shot."""

== Overview ==

*References* or *Payloads* an AYON USD product into the current USD layer.

This node allows you to select which AYON USD product to load. It does this by utilizing a [Node:lop/reference] to load the specified AYON USD product.

@parameters 
~~~ Choose Product ~~~
Project:
    The name of the AYON project from which to load products.
Folder Path:
    The path to the AYON folder (entity) within the project's hierarchy.
Product Type: 
    The type of AYON product to load (e.g., USD, point cache, camera, etc.).
    :note:
        Product type can have multiple representations.
Version:
    The specific version of the product you want to load.
Representation:
    The name of the representation, which refers to the file format (e.g., USD, USDA, ABC).
    :note:
        we use the representation name to query the file path from AYON.
Refresh:
    Click to refresh and retry applying the product load parameters to load the correct file
Reload Files:
    Click to reload the contents of all files imported by this node. This also clears the cache of file wilrdcard pattern expansions.
File:
    The file path that will be loaded. This is locked by default as it is typically generated by the node.
    :tip:
        It's locked by default as it should be computed by the node.
Primitive Root:
    The referenced prim will be overlayed onto this prim, and the referenced prim’s descendants will become this prim’s descendants. If this prim doesn’t exist, the node will create it.
```
how it looks like:
![image](https://github.com/user-attachments/assets/85ffee30-7f54-4bdf-8b59-1fb803573c30)

> [!Note]
> I don't know how and why it shows the icon although I reverted `#icon` to the current status of the PR.

Reply:
@MustafaJafar by the way - why does the icon suddenly not work for you anymore? :)